Succulent/Classes/Succulent.swift

```diff
@@ -9,20 +9,36 @@
 import Embassy
 import Foundation
 
+public struct Configuration {
+    public var port: Int?
+    public var ignoreParameters: Set<String>?
```

Comment:
Documentation:

> A set of query string parameter names that should be ignored for the purposes of matching incoming requests to recorded requests.

Reply:
Let's change this to an array of strings and convert it to a set in our `init(Configuration)` function.